Makefile

```diff
@@ -88,11 +88,14 @@ GO_PACKAGE_PATH ?= github.com/${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}
 
 GIT_COMMIT_ID = $(shell git rev-parse --short HEAD)
 
-OPERATOR_VERSION ?= 0.0.10
+OPERATOR_STABLE_VERSION ?= 0.0.11
+OPERATOR_DEV_VERSION ?= $(OPERATOR_STABLE_VERSION)-dev
```

Comment:
To keep things simple, you could hardcode the dev channel to have always have the image tagged `latest`?

Reply:
I could hardcode the `latest` tag to the `dev` operator image in the CSV but the `OPERATPR_DEV_VERSION` is used to version the `dev` CSV and the operator release as well. 

AFAIK we cannot use `latest` as a release when pushing the operator package to the quay app registry, it does not have the "override" feature, so when we try to push the second time, it fails complaining that the `latest` release already exists.